vignettes/getting.Rmd

```diff
@@ -340,12 +340,12 @@ round(sum(routes_towns$foot + routes_towns$bicycle) / sum(routes_towns$all) * 10
 ```
 
 Currently, only around 34% of commute trips in the region's 'town' areas are made by walking and cycling (27% across all zones in North Yorkshire, and a much lower proportion in terms of distance).
-We explore this in more detail by looking at the relationship between trip distance and mode share, in Figure \@ref(fig:distmode) (left).
+We explore this in more detail by looking at the relationship between trip distance and mode share, as shown in Figure \@ref(fig:distmode) (a).
 We will create a scenario representing the outcome of policies that incentivise people to replace car trips with walking and cycling, focussing on the red boxes in Figure \@ref(fig:distmode).
 The scenario will replace 50% of car trips of less than 1 km with walking and 10% of trips between 1 km and 2 km in distance.
-The remaining car trips will be replaced by cycling, with the percentages of trips that switch for each OD determined by the uptake function in the Go Dutch Scenario of the PCT, resulting in the right graphic shown in Figure \@ref(fig:distmode).
+The remaining car trips will be replaced by cycling, with the percentages of trips that switch for each OD determined by the uptake function in the Go Dutch Scenario of the PCT, resulting in the graphic shown in Figure \@ref(fig:distmode) (b).
 
-```{r distmode, echo=FALSE, fig.cap="Relationship between distance (x axis) and mode share (y axis) in towns and cities in North Yorkshire.", out.width="49%", fig.show='hold', message=FALSE}
+```{r distmode, echo=FALSE, fig.cap="Relationship between distance (x axis) and mode share (y axis) in towns and cities in North Yorkshire. (a) left: existing mode shares; (b) right: mode shares under high active travel uptake scenario.", out.width="49%", fig.show='hold', message=FALSE}
```

Comment:
Great improvement to the original training doc. I'll aim to port the other changes across for the benefit of users of the pct package.